build.sh

```diff
@@ -125,14 +125,28 @@ pyinstaller main.pyw --onefile --clean --noconfirm \
     --add-data "data/*:data" \
     --name "imd-gui" \
     --icon "data/jukebox_256.ico" \
-    --distpath bin 2>tmp/latest.log \
+    --distpath bin 2>build/latest.log \
 
-#Print success message
-if tail -1 tmp/latest.log | grep successfully >/dev/null; then
+#Determine if build has succeeded or failed
+if tail -1 build/latest.log | grep successfully >/dev/null; then
     printf "%b %bBuild succeeded!%b\\n" "${TICK}" "${COL_LIGHT_GREEN}" "${COL_NC}"
 else
     printf "%b %bBuild failed!%b\\n" "${CROSS}" "${COL_LIGHT_RED}" "${COL_NC}"
 
+    #Ask the user if he wants to view the log
```

Comment:
I think the user will get more value if they can see the log from pyinstaller in-console. That way they know pyinstaller is making progress, see errors in real-time, etc.

I would recommend sending stderr to stdout, then piping that to `tee` so pyinstaller can log to the console and to a file. See above

Reply:
Taken care of.